src/GovUk.Education.ExploreEducationStatistics.Common/BlobContainers.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace GovUk.Education.ExploreEducationStatistics.Common
+{
+    public interface IBlobContainer
+    {
+        public string Name { get; }
+        public string EmulatedName { get; }
+    }
+
+    public static class BlobContainers
+    {
+        public static readonly BlobContainer PrivateReleaseFiles = new BlobContainer("releases");
+        public static readonly BlobContainer PublicReleaseFiles = new BlobContainer("downloads");
+        public static readonly BlobContainer PublicContentContainerName = new BlobContainer("cache");
```

Comment:
Might be good to rename this in the future?

Reply:
Good spot, I didn't rename this before to reduce the changes in the diff which was already going to be big. I've added it to a tidy up task which is EES-1936 which will include other stuff like removing the migration database fields.